bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RefreshingOAuth2CredentialsInterceptor.java

```diff
@@ -90,24 +91,30 @@
 
     final IOException exception;
     final String header;
-    final long staleTimeMs;
-    final long expiresTimeMs;
+    final Long staleTimeMs;
```

Comment:
can you document that when staleTimeMs is null, the cache will be stale or expired?